### ISSUE TITLE
Consistent gitsigns colors

### ISSUE THREAD

**Issue by lixvbnet** [NvChad/base46#82]
When switching themes, gitsigns colors are changing too. Some themes mark added lines green but others might be blue. 

It would be great if you could make gitsigns colors more consistent regardless of theme choices, green for added lines, blue for updated, red or grey for deleted, for example. (Also what I’ve got used to :-)

**Comment by siduck** [comment]
that depends on the theme! for example green is a dominant color in everforest then we use green in it but blue in onedark and so on, some light themes might not look good with blue gitsign colors as well

**Comment by lixvbnet** [comment]
@siduck yeah I’m planning to do so. I haven’t read the docs for customizing themes, but now it’s definitely the time to do this :-)  thanks!
